proxa/Docusign-Integration
# Added authorization code grant auth with DocuSign API

**Test Procedures**
1. `gradle bR`
2. Navigate to http://localhost:8080
3. Click one of the buttons
4. Log in with your Docusign account
5. Verify that you are redirected back to the app's original URL but with an access_token (you should see some JSON in the browser)
